package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dapr-client",
+    "name": "@dapr/dapr",
     "version": "2.2.0-rc.0",
```

Comment:
Note: We need to delete this from npm before running the workflow. Otherwise, it won't be able to overwrite the same version.